public/api/1.0/openapi.yaml

```diff
@@ -57,7 +65,248 @@ components:
           value: Bermuda
           properties:
             actionIdentification: "824"
-  
+    CustomsOffice:
+      type: object
+      x-tags:
+        - CustomsOffices
+      required:
+        - referenceNumber
+        - activeFrom
+        - countryCode
+        - postalCode
+        - traderDedicated
+        - customsOfficeSpecificNotesCodes
+        - customsOfficeLsd
+        - customsOfficeTimetable
+      additionalProperties: false
+      properties:
+        referenceNumber:
+          type: string
+          description: The unique reference number for a customs office.
+        activeFrom:
+          type: string
+          format: date-time
+          description: The time from which the current office is active.
+        activeTo:
```

Comment:
I'm not certain that we should provide this field in the response - similar to how we have a cut-down JSON formatter for codelist entries.